services/src/error.rs

```diff
@@ -393,6 +393,14 @@ pub enum Error {
     OidcError {
         source: crate::pro::users::OidcError,
     },
+
+    #[snafu(display(
+        "Could not resolve a Proj string for this SpatialReference: {}",
+        spatial_ref
+    ))]
+    ProjStringUnresolvable {
+        spatial_ref: SpatialReference,
+    },
```

Comment:
I'm not sure that this should be an error at the root level. Could you put it where it happens?

Reply:
why not? This is a user input and they should should be reported if they input it wrong. Do you want me to wrap it in a WCS error? "WCS Error: Could not resolve a Proj string for this SpatialReference:"? The user should know that they are in a wcs request though. It's not like that this is some deep internal error.

Reply:
it happens in crate::api::model::datatypes. I don't think it would be any better to have a separate ApiError I guess so I would leave it here. Do you have a suggestions where to put it?